src/extension.ts

```diff
@@ -65,16 +65,21 @@ async function checkAngularVersions(quiet = false) {
       .showInformationMessage(
         `Your version of Angular is outdated.\r\nCurrent version: ${
           coreOutdated.currentVersion
-        }\r\nNew version: ${coreOutdated.newVersion}`,
+        }\r\nLatest version: ${coreOutdated.newVersion}\r\nNext Version: ${
+          coreOutdated.nextVersion
+        }`,
         { modal: true },
-        'Update Angular',
-        'Not now'
+        'Update to Latest',
+        'Update to vNext(risky)'
       )
       .then(async value => {
-        if (value !== 'Update Angular') {
+        if (!value) {
+          return
+        }
+        if (value && !value.includes('Update')) {
           return
         } else {
-          await doAngularUpdate()
```

Comment:
@kentonbmax was this change intentional? I may be wrong but it looks like this switch means that we no longer check for a clean git before running update commands.

Reply:
FYI, I will push a fix in #34 (specific commit for fix is https://github.com/duluca/angular-evergreen/pull/34/commits/8421a80c25c70911f8066a404d893799b518f6a0).